### ISSUE TITLE
Proposal: Merge Starter Packs and Lists

### ISSUE THREAD

**Issue by knotbin** [bluesky-social/social-app#5940]
Lists and starter packs offer the same functionality - adding accounts to a collection - for different purposes. However, their features are fragmented even though they are very similar. Lists can be pinned to the home feed while starter packs cannot, people can be easily added to lists from their profile, while they can't be easily added to starter packs, lists are searchable while starter packs are not.

This proposal argues for the merging of the two. Starter packs would be removed in favor of lists, which would now have an option to "Share as a Starter Pack"

Current starter packs would be converted to lists.
The only obstacle to this is the fact that lists can contain feeds. Two options for this: 
1. Get rid of feeds in current starter packs, as they are used in very few starter packs, and of the starter packs they are used on, they are a relatively unimportant addition.
2. Allow lists to contain feeds

This proposal will simplify an overcomplicated and confusing lineup of user collection features that are needlessly separated and fragmented. Please let me know your feedback.

**Comment by wlonkly** [comment]
I am a fan of this plan. Lists feel native and dynamic, and starter packs bolted on and one-time, but the popularity of starter packs has been a great boost for the community. 

But I noticed what I think might be an error in the proposal:

> The only obstacle to this is the fact that lists can contain feeds

"that lists cannot contain feeds", right?

**Comment by svivian** [comment]
Lists *are* feeds though. Or, nobody has managed to articulate any difference between them.

**Comment by wlonkly** [comment]
> Lists _are_ feeds though. Or, nobody has managed to articulate any difference between them.

No, they're not really that related. A List is a pre-configured list (sorry) of users, which can be used as a "show me these users' posts" list like lists on Twitter did, or as a block list. 

A feed is an interface where a service, external to the main parts of Bluesky, can apply whatever logic it wants to include certain posts.  That might be "posts from these users", where it works roughly like a list does (but maybe "these users" changes somehow), or it can be based on some qualities of the posts themselves (like "[popular in your network](https://bsky.app/profile/bsky.app/feed/with-friends)" or "[contains the name of a city](https://bsky.app/profile/did:plc:3v55yprp3owcom57wrosvf5i/feed/aaai72gpp25t6)" or "[is getting ratio'd](https://bsky.app/profile/did:plc:pmyqirafcp3jqdhrl7crpq7t/feed/aaapfo6jminv2)".) 

Feeds are more like "pluggable algorithms", but they also require someone to run the software that implements the algorithm on their own server somewhere.

Starter Packs are a lot like lists because they both are a List (again, sorry) of users, but Starter Packs (unlike Lists) can also refer to Feeds. If they were to merge Starter Packs and Lists, then they'd have to do something about the fact that Lists can't refer to Feeds.

**Comment by svivian** [comment]
OK thanks for the explanation. So are there starter packs that contain different types of Feeds? Every one that I’ve seen, the Feed was just all the posts by those users.

**Comment by wlonkly** [comment]
> OK thanks for the explanation. So are there starter packs that contain different types of Feeds? Every one that I’ve seen, the Feed was just all the posts by those users.

Yeah, if you go through the process of creating a starter pack [here](https://bsky.app/starter-pack/create), you can see that you can use it to recommend both Users and Feeds to people. In practice, though, everyone seems to be using them for Users. 

(Keep in mind that the "Posts" tab at the top of a starter pack isn't a Feed, it's just a feature of Starter Packs. You can't add that to your Feeds, for instance.)

Interestingly, the [announcement blog post](https://bsky.social/about/blog/06-26-2024-starter-packs) listed Feeds first!

**Comment by cooljeanius** [comment]
See further discussion in #5970
